pkg/scaling/resolver/scale_resolvers.go

```diff
@@ -40,6 +43,22 @@ const (
 	referenceCloser   = ')'
 )
 
+var kedaNamespace, _ = util.GetClusterObjectNamespace()
+
+// isSecretAccessRestricted returns whether secret access need to be restricted in KEDA namespace
+func isSecretAccessRestricted(logger logr.Logger) bool {
+	const RestrictSecretAccessEnvVar = "KEDA_RESTRICT_SECRET_ACCESS"
+	restrictSecretAccess, found := os.LookupEnv(RestrictSecretAccessEnvVar)
+	if !found {
+		return false
+	}
+	if strings.ToLower(restrictSecretAccess) == "true" {
+		logger.V(1).Info("Secret Access is restricted to be in KEDA namespace, pls. use ClusterTriggerAuthentication instead of TriggerAuthentication", "Env Var", RestrictSecretAccessEnvVar, "Env Value", strings.ToLower(restrictSecretAccess))
```

Comment:
```suggestion
		logger.V(1).Info("Secret Access is restricted to be in KEDA namespace, please use ClusterTriggerAuthentication instead of TriggerAuthentication", "Env Var", RestrictSecretAccessEnvVar, "Env Value", strings.ToLower(restrictSecretAccess))
```

We should print here the specific namespace instead of `KEDA namespace`.

Also, I don't think it is necessary, to call the `os.LookupEnv()` everytime this function is called, I think it is enough to validate this ENV during startup once. Could you please refactor this? Maybe in `util.GetClusterObjectNamespace()` or maybe somewhere else?

Reply:
Good suggestion, update, thanks!